{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/people/models.py

```diff
@@ -143,5 +143,9 @@ def twitter_url(self):
         if self.twitter:
             return f'https://twitter.com/{self.twitter}'
 
+    @property
```

Comment:
Use `django.utils.functional.cached_property` if you're not expecting this to change over the lifetime of the instance.